src/terra-dev-site/doc/Configuration.b/page.config.c.doc.md

```diff
@@ -0,0 +1,79 @@
+# Page Config
+
+Page config is generally generated for you using the generatePages config ins the site.config, but if you need or want custom page config you can provide your own. Below is an example page config. The config is sorted by Pagetype with ordered arrays of child pages.
+
+```javascript
+const config = {
+  doc: [
+    {
+      name: 'Getting Started',
+      path: '/getting-started',
+      content: '../src/terra-dev-site/doc/gettingStarted.md',
+      type: 'md'
```

Comment:
I think need to explain what each key means and when/what needs included. For example,

- doc & test are the pageTypes
- what is `type` on each page hash? is it needed. Can we not grab that from the file content path?
- do we need the extension in the content path? I notice two examples email accidentally? dropped it? [ex 1](https://github.com/cerner/terra-dev-site/pull/77/files#diff-fff84418a779eaf8259a4a61c96d40d3R63) & [ex 2](https://github.com/cerner/terra-dev-site/pull/77/files#diff-fff84418a779eaf8259a4a61c96d40d3R73)